app/models/backend_api.rb

```diff
@@ -34,6 +34,16 @@ class BackendApi < ApplicationRecord
 
   scope :orphans, -> { where.has { id.not_in(BackendApiConfig.selecting { :backend_api_id }) } }
 
+  scope :without_service, (lambda do |service_id|
```

Comment:
```suggestion
  scope :not_used_by, (lambda do |service_id|
```

Reply:
`not _used_by` lacks a what :smile: like: `not_used_by_product` :thinking: 
WDYT @guicassolato ?